src/main/java/com/syncapse/jenkinsci/plugins/awscloudformationwrapper/ParametersConverter.java

```diff
@@ -0,0 +1,26 @@
+package com.syncapse.jenkinsci.plugins.awscloudformationwrapper;
```

Comment:
I see a unit test for the parser, but I didn't see one for this class. Did I miss it?